src/platforms/common/blackpill-f4/blackpill-f4.c

```diff
@@ -73,26 +74,19 @@ void platform_init(void)
 #endif
 	rcc_clock_setup_pll(&rcc_hse_25mhz_3v3[PLATFORM_CLOCK_FREQ]);
 
-	/* Enable peripherals */
-	rcc_periph_clock_enable(RCC_OTGFS);
```

Comment:
We see this being dropped from here, but not added back elsewhere - why is this correct?

Reply:
Sorry for not copying data out of the commit message so that it's visible to you.
* Drop `RCC_OTGFS` enabling because it happens in `stm32f107_usbd_init()`

https://github.com/libopencm3/libopencm3/blob/1f3abd43763fa39d23e737602b6d0011a45c70b2/lib/usb/usb_f107.c#L56
I am not asking the question why all the other F4/dwc platforms repeat the mantra. Maybe `libopencm3` didn't do this prior to 2016 is one possible reason. We're tracking current libopencm3.

Reply:
Ahhhh, ok, that makes much more sense then - yeah, letting locm3 do its job and not repeating that in the platform makes total sense.